src/solver/conv_hip_implicit_gemm_fwd_v4r4_xdlops.cpp

```diff
@@ -465,6 +500,19 @@ PerformanceImplicitGemmForwardV4R4Xdlops::CalculateGemmBBlockCopyPerformancePara
         // vector write into LDS
         DstDataPerWrite_GemmKPack = gcd(DstDataPerWrite_GemmKPack, data_per_thread_copy_gemmkpack);
 
+        if(miopen::IsEnabled(
+               MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_FWD_V4R4_XDLOPS_ADD_VECTOR_LOAD_GEMMN_TUNE_PARAM{}))
+        {
+            if(ctx.IsFp16())
+            {
+                if((SrcDataPerRead_GemmN > 1) &&
```

Comment:
What does this logic means? 
What is the purpose of this logic?
If we remove this logic, the code would not compile or it would run slow?

Reply:
@asroy , please help to review, tunning is disable for non-fp16, and fp16 tunning filterring  is moved to IsFastToBeUsedForTuning, thanks.